pkg/command/feature.go

```diff
@@ -3,18 +3,21 @@ package command
 import (
 	"context"
 	"fmt"
+	"path/filepath"
 	"regexp"
 	"strings"
+	"time"
 
 	"github.com/spf13/afero"
+
 	"github.com/xenitab/gitops-promotion/pkg/config"
 	"github.com/xenitab/gitops-promotion/pkg/git"
 	"github.com/xenitab/gitops-promotion/pkg/manifest"
 )
 
 // FeatureCommand is similar to NewCommand but creates a PR with a temporary deployment of the application.
```

Comment:
I think it should say `FeatureNewCommand` in the comment. Surprised that the linter don't say anything about this.